cypress/fixtures/QiCoreHRCompare/GroupScoringAndPCSections.html

```diff
@@ -30,6 +30,15 @@
 
 
 
+  
```

Comment:
Other than this one, no file contents were changed. All the other files were just moved.